ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -166,7 +166,9 @@ std::shared_ptr<Function> FrontEndPDPD::convert_each_node(
 
             if (!named_outputs.empty()) {
                 // set layer name by the name of first output var
-                const auto& tensor_name = op_desc.outputs().begin()->arguments()[0];
+                std::string tensor_name = "None";
```

Comment:
The fix is not correct. If node doesn't have tensor name we shouldn't set it for node.
Just skip it.

@mvafin @itikhono I think we also need to set operation tensor names

Reply:
have changed to skip these none tensor name.
@mvafin @itikhono @ilyachur , about 'set op tensor names', do you mean we just need keep 'ng_outputs[idx].get_tensor().set_names({var_name});'  as before?

Reply:
what will happen if an output of a op has no name?

Reply:
It looks to me this is a case when 1st port of operation is not used (op_desc.outputs().begin()). In that case, I think we can either not assign a name to the operation as it was done in this PR or walk through other ports and set the name to the first seen name in ports. Either way, this won't result in the tensors without names, because there won't be a tensor on this place. We will only have an operation without a name, which is not a big problem.